client/web/compose/src/views/Admin/Pages/Builder.vue

```diff
@@ -779,7 +779,7 @@ export default {
 
       layout.meta.title = `${layout.meta.title}`
 
-      this.createPageLayout(this.layout).then(({ layoutID }) => {
+      this.createPageLayout(layout).then(({ layoutID }) => {
```

Comment:
Why this layout? The layout is created three lines above, and that one should be used

Reply:
This is a case of me being blind, all good